dotnet-developer-projects.md

```diff
@@ -107,6 +108,8 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [Automatonymous](https://github.com/MassTransit/Automatonymous) - An asynchronous state machine engine with messaging support. 
   * [Bing.RestClient](https://github.com/AdvancedREI/Bing.RestClient) - Access the Bing REST APIs from a unified client.
   * [BoxKite.Twitter](https://github.com/NickHodge/BoxKite.Twitter/) - Twitter .NET Client Library for the 1.1 Twitter API, incorporating REST API, User streaming and Search Streaming. Uses Reactive Extensions (Rx).
+  * [Castle.LoggingFacility.MsLogging](https://github.com/volosoft/castle-logging-ms-adapter) - Castle Logging Adapter for Microsoft.Extensions.Logging Package
```

Comment:
Package --> package. (not that this should end with a period)]
Can you provide any additional detail?